app/views/reservations/_form.html.erb

```diff
@@ -24,6 +24,26 @@
     </div>
   </div>
 
+  <% if @reservation.checked_out %>
+  <div class="form-group">
+    <%= f.label :checked_out, 'Check-out date and time', class: 'col-sm-2 control-label' %>
```

Comment:
We can just label this 'Checked Out' - I don't think we need to specify 'date and time'.